extensions/amp-mega-menu/0.1/amp-mega-menu.css

```diff
@@ -0,0 +1,74 @@
+/**
```

Comment:
Does everything in this file need to be `!important`? (Genuinely asking, not intimately familiar with CSS conventions in `amphtml`

Reply:
I think the process is usually start off by making the properties non-overridable, then release any that has a need for user customization.